### PR TITLE
add check for adb 

### PR DESCRIPTION
**Pull request by DG12**
I know it's a kind of special test script but 
I added a test for adb (Android Debug Bridge) presence
and 
let user know that stiles will be wget ed from release

### CODE REVIEW

**Review by DG12 — COMMENTED** [review]
I tried to delete this tar from my repository but got a 404!

File: system_tests/test_sdk_dfu.sh [hunk 1]
```diff
@@ -17,6 +21,13 @@ rm *dfu_app.zip*
 rm *dfu.zip*
 rm *full.hex*
 
+pwd
+echo  Will  get:
+echo "${TAG}/"
+echo "          ${TEST_VARIANT}"
+echo "          ${APP_VARIANT}"
+echo "          ${SDK_VARIANT}                      from github  OK?"
```
Comment (ojousima) [inline system_tests/test_sdk_dfu.sh:29]: This check should be removed.

**Review by ojousima — CHANGES_REQUESTED** [review]

**Comment by ojousima** [comment]
ok to test

File: system_tests/test_sdk_dfu.sh [hunk 2]
```diff
@@ -17,6 +21,13 @@ rm *dfu_app.zip*
 rm *dfu.zip*
 rm *full.hex*
 
+pwd
```
Comment (ojousima) [inline system_tests/test_sdk_dfu.sh:24]: This chunk should be removed, not only commented out

**Review by ojousima — CHANGES_REQUESTED** [review]

**Review by ojousima — APPROVED** [review]
Thanks, will merge once CI passes

**Pull request merged**